jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -87,4 +92,22 @@ class Config:
             env_file = Path(".env")
 
 
-settings = Settings()
+def build_settings(*args, **kwargs):
+    """
+    Return a Setting object and handle ValidationError with a message to the user.
+    """
+    try:
+        return Settings(*args, **kwargs)
+    except ValidationError:
+        terminal_message(
+            conjoin(
+                "A configuration error was detected.",
+                "",
+                f"[yellow]Please contact [bold]{OV_CONTACT}[/bold] for support and trouble-shooting",
```

Comment:
isn't needed to close the `[yellow]` statement?

Reply:
Absolutely, I will commit the fix